.github/workflows/size.yml

```diff
@@ -8,18 +8,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - name: compressed-size-action
         uses: preactjs/compressed-size-action@v2
         with:
           pattern: 'dist/!(*.js.map)'
           build-script: build
           repo-token: '${{ secrets.GITHUB_TOKEN }}'
-
-      - name: compressed-size-action
-        uses: preactjs/compressed-size-action@v2
-        with:
-          pattern: 'test/fixtures/**/dist/!(*.map)'
-          build-script: 'test'
-          repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
I don't know that this is particularly useful, any size difference will be accounted for in the tests already.